Dockerfile

```diff
@@ -1,10 +1,13 @@
 FROM debian:jessie
 
 ENV ARCH=x86_64-unknown-linux-gnu
-ENV RUST_RELEASE=1.9.0
-ENV LLVM_RELEASE=3.9
+ENV RUST_RELEASE=1.13.0
+ENV LLVM_RELEASE=4.0
```

Comment:
We are using LLVM 3.9 so far. See https://github.com/tagua-vm/llvm/blob/57e3d460b24a02269f9bc3edc8fd666202279536/.travis.yml#L41.